server/src/main/java/io/littlehorse/server/streams/topology/core/processors/CommandProcessor.java

```diff
@@ -140,7 +140,12 @@ private ProcessorExecutionContext buildExecutionContext(Record<String, Command>
     }
 
     public void onPartitionClaimed() {
+        if (partitionIsClaimed) {
+            throw new RuntimeException("Re-claiming partition! Yikes!");
```

Comment:
This looks like I wrote this code :joy: You are picking up my style!

Reply:
That's your code. just moving